FIAT/kong_mulder_veldhuizen.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (C) 2008 Robert C. Kirby (Texas Tech University)
+#
+# contributions from Keith Roberts (University of Sao Paulo)
+#
+#
+# This file is part of FIAT (https://www.fenicsproject.org)
+#
+# SPDX-License-Identifier:    LGPL-3.0-or-later
+from FIAT import (
+    finite_element,
+    dual_set,
+    functional,
+    Bubble,
+    FacetBubble,
+    Lagrange,
+    NodalEnrichedElement,
+    RestrictedElement,
+)
+from FIAT.quadrature_schemes import create_quadrature  # noqa: F401
+
+
+def _get_topology(ref_el, degree):
```

Comment:
Better name?  topology refers to the cell's connectivity frequently.  This is a helper function to get the connection of the points/dofs to the cell entities.  Following rest of FIAT, `_get_entity_ids` maybe?

Reply:
I altered the name to the one you suggested.